docs/env.md

```diff
@@ -63,6 +63,10 @@ Below is a list of currently supported environment variables:
 - **Required**
 - default: `https://web.opensrp-stage.smartregister.org/oauth/state`
 
+11. **REACT_APP_SENTRY_DSN**
```

Comment:
@HaseemUlHaq assigned you this: https://github.com/OpenSRP/web/issues/186

Reply:
Will address this in a separate branch covering issue #186